CONTRIBUTING.md

```diff
@@ -351,6 +352,15 @@ we have scripts that you can use, like `mage StartDockerRegistry` or
 
 Slow! 🐢 This takes between 8-16 minutes, depending on your computer hardware.
 
+🚧 We are in the process of updating our integration tests to reduce total test time by combining test cases into larger tests so that we aren't running expensive set up operations like building bundles as often.
+The goal is to make the integration tests use the same test framework [tests/tester] which tests porter using the CLI instead of by calling porter's code directly.
+Follow the example of the [hello smoke test] for a good test to copy and make new tests from.
+We use `require` instead of `assert` in these long tests because we want the test to stop on the first failure instead of having the rest of the test output a bunch of failures too.
```

Comment:
I think this should explain what `require` and `assert` are:

```suggestion
We use the [Testify library](https://github.com/stretchr/testify)'s [`require`](https://pkg.go.dev/github.com/stretchr/testify@v1.8.2/require) package (instead of [`assert`](https://pkg.go.dev/github.com/stretchr/testify@v1.8.2/assert)) in these long tests because we want the test to stop on the first failure instead of having the rest of the test output a bunch of failures too.
```